apps/web-giddh/src/assets/css/color.scss

```diff
@@ -124,4 +125,13 @@ $linen: #fff3e8;
 $tutu: #ffe5e5;
 $denim: #2c6ebd;
 $lavender-blue: #bdc3ff;
-$medium-gray: #ebeef0
\ No newline at end of file
+$medium-gray: #ebeef0;
+
+/* new design colors */
+$black-pearl: #212528;
+$trout: #494D50;
+$solitude: #eceef0;
```

Comment:
@prateek-giddh solitude color is already defined in line no. 70. Please remove if these two colors has less difference or add some other variable name for this color.

Reply:
Done